scripts/create_tables.sql

```diff
@@ -1 +1,29 @@
 -- create doctors, patients, and appointments tables
+CREATE TABLE doctors(
+  id SERIAL PRIMARY KEY,
+  family_name TEXT,
+  given_name TEXT,
+  gender TEXT,
+  born_on DATE,
+  specialty TEXT
+);
+
+CREATE TABLE patients(
+  id SERIAL PRIMARY KEY,
+  family_name TEXT,
+  given_name TEXT,
+  gender TEXT,
+  born_on DATE,
+  insurance TEXT
+);
+
+CREATE TABLE appointments (
+  id SERIAL PRIMARY KEY,
+  appointment_date DATE,
```

Comment:
Okay, I wasn't sure because sometimes in the labs if I named something 'date' it would turn purple like a keyword, so lately I've been trying to avoid using the keywords as column names